plasmapy/diagnostics/tests/test_langmuir.py

```diff
@@ -1,40 +1,86 @@
-"""Tests for functions that use Langmuir analysis functions."""
+# coding=utf-8
+"""Tests for Langmuir probe analysis functions."""
 
 import numpy as np
+from astropy import units as u
+import pytest
+import plasmapy.constants as const
```

Comment:
I think we usually import the constants into the local namespace (though that may not be good practice).

Reply:
Yeah, it might not be. Based on what I tried with util docs, if you try to use autodoc with, say, defaultdict imported from collections, that's going to register as an actual thing in that module, while if you just import collections it won't show up.

I think we shouldn't press consistency too hard here. Do whatever works!

Reply:
Yup, just bringing it up as a point of discussion. In my personal codes I tend to use import as instead of importing into the local namespace.
Perhaps we should open this as an issue and find a good way of doing this so that the docs are happy and that we aren't overwriting any variables (like `e` the electric charge vs. `e` the natural number.)